YunoHost/yunohost-admin
# Enh permissions (new selectize + confirmation for ssh/sftp)

- add a new selectize component (replacing two formers) based on bootstrap-vue component (so should be accessible)
- add a confirmation modal before add ssh/sftp permission to a group (Should fix https://github.com/YunoHost/issues/issues/1775)
- fix a cache issue related to group permissions

# PR STATUS
Tested on my side on a broken instance ; )
Had to modify a bit the store to fix the cache issue + had to rebase tricky parts, hope i didn't break anything.

# HOW TO TEST
- add/remove permissions or users in `/users/permissions` to see if it's still working
- add ssh/sftp permissions to get the confirmation modal
